addons/website/static/src/scss/website.scss

```diff
@@ -1458,3 +1458,20 @@ $ribbon-padding: 100px;
     // hide them if a modal is open.
     display: none !important;
 }
+
+// web/bootstrap_review.scss makes the card's background white
+// no matter what. We need to adjust theme colors for that
+.o_portal main {
+    a:not(.btn), .btn-link {
+        $portal-link-color: increase-contrast(o-color('o-cc1-link'), $card-bg);
+        color: $portal-link-color;
+        &:hover {
+            color: increase-contrast($portal-link-color, $card-bg);
+        }
+    }
+}
+.o_portal_wrap {
+    .navbar-light {
+        background-color: $card-bg;
+    }
+}
```

Comment:
I don't see how this is related to portal. Like you clearly say in your code comment, you seem to adapt the link colors inside of cards... what's the difference with the `table` rule you added in bootstrap_review? (And why would this portal-related rule be defined in website?) Also in any case `o-color('o-cc1-link')` does not really make sense, you can use `$link-color` which is the final color of the global website links.

Reply:
Actually BVR already told you that if I understand
![image](https://user-images.githubusercontent.com/10338094/156024314-dafa4b50-b55b-4498-b7b0-95e5ab78db8b.png)
Was that conversation really meant to be resolved?